account/utils.go

```diff
@@ -14,6 +16,54 @@ var (
 	ErrInvalidMnemonicPhraseLength = errors.New("invalid mnemonic phrase length; valid lengths are 12, 15, 18, 21, and 24")
 )
 
+type LoginParams struct {
+	ChatAddress    common.Address   `json:"chatAddress"`
+	Password       string           `json:"password"`
+	MainAccount    common.Address   `json:"mainAccount"`
+	WatchAddresses []common.Address `json:"watchAddresses"`
```

Comment:
chat address, main account and watch addresses are stored in status-react in current version? @flexsurfer @yenda 

can you please specify what else is stored in status-react related to multiaccount?

Reply:
currently only chat address and main account stored in status-react

Reply:
where should i get watch addresses then?

i am trying to understand how this flow will change when multiaccount db state will be in status-go

Reply:
I think we are still not using the multi-account functions. 
Before your PR I expect status-react to derive and store new addresses and send them as watchAddresses during login. together with other watch only addresses.

Reply:
I'm not a big fan of the terminology here, it's going to need some changes but probably not in this PR because we don't have the time for that.

I think WatchAddresses here is refering to the list of accounts that status-react is passing to status-go that need to be watched by the wallet service.
This should be simply called accounts. Each multiaccount contains X accounts. Chat accounts and wallet accounts are technically the same thing it's just that one is loaded in memory and not the other.

@dshulyak for status-go db I would use an account db (but not the one for the user accounts, in status-react we have sorted this out by calling user accounts multiaccounts to avoid any confusion, ideally status-go should do the same or find a similar solution) and have some attributes like "load-at-startup?", "watch-only?" to distinguish these different types of accounts wdyt?